### PR TITLE
AzureStack Fabric Admin Specification

### PR DESCRIPTION
**Pull request by deathly809**
The specification has been regenerated.  

### PR information
- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [X] If applicable, the PR references the bug/issue that it fixes.
- [X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [X] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X] My spec meets the review criteria:
  - [X] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [X] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by deathly809** [comment]
Regenerating PR.  This one is broke.

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/stable/2016-05-01/Fabric.json [hunk 1]
```diff
@@ -1,107 +1,214 @@
 {
- "swagger": "2.0",
```
Comment (sergey-shandar) [inline specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/stable/2016-05-01/Fabric.json]: @deathly809 could you revert formatting so it is to review?

**Review by sergey-shandar — CHANGES_REQUESTED** [review]

**Comment by sergey-shandar** [comment]
@ravbhatnagar changes in a stable API.

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/stable/2016-05-01/Fabric.json (hunk 1 above)
Comment (deathly809) [reply specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/stable/2016-05-01/Fabric.json]: I don't understand what you are asking.

**Comment by ravbhatnagar** [comment]
This is Azure Stack RP, correct? I am not sure if ARM needs to review this. @deathly809 - Please let me know.

**Comment by ravbhatnagar** [comment]
Removing ARM feedback label for now. Please let me know if ARM needs to review which I dont think we need to.

**Comment by deathly809** [comment]
@ravbhatnagar  Just for Azure Stack, Azure does not have an admin interface.

File: specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/stable/2016-05-01/Fabric.json (hunk 1 above)
Comment (sergey-shandar) [reply specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/stable/2016-05-01/Fabric.json]: Sorry, I mean "it's easy to review". It's really hard to review if there are a lot of formatting changes, such as replacing indent from 4 symbols to 1 symbol. Please, have a look the file `Fabric.json` in https://github.com/Azure/azure-rest-api-specs/pull/2809/files

Comment (deathly809) [reply specification/azsadmin/resource-manager/fabric/Microsoft.Fabric.Admin/stable/2016-05-01/Fabric.json]: Then some of my files with be 2 spaces and the others 4 spaces.  Would this be a temporary formatting change and then I could change it back?  I really don't want some files being 2 and some being 4.

**Comment by sergey-shandar** [comment]
Closing the PR. @deathly809 you can create a new one and let me know.

**Pull request closed without merging**